2nd-gen/packages/swc/.storybook/DocumentTemplate.mdx

```diff
@@ -28,26 +29,18 @@ This template will likely be going away in the future, but for now it's here to
         <Description />
         <Primary />
         <Controls />
```

Comment:
I would save the controls for last. Just let them see the component and some overall info about when to use.